src/components/dashboard/users/AccessContainer.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FunctionComponent } from 'react';
+import globalize from '../../../scripts/globalize';
+import CheckBoxElement from './CheckBoxElement';
+
+type IProps = {
+    ContainerClassName?: string;
+    HeaderTitle?: string;
+    CheckBoxClassName?: string;
+    CheckBoxTitle?: string;
+    ListContainerClassName?: string;
+    AccessClassName?: string;
+    ListTitle?: string;
+    Description?: string;
+    children?: React.ReactNode
+}
+
+const AccessContainer: FunctionComponent<IProps> = ({ContainerClassName, HeaderTitle, CheckBoxClassName, CheckBoxTitle, ListContainerClassName, AccessClassName, ListTitle, Description, children }: IProps) => {
+    return (
+        <div className={ContainerClassName}>
+            <h2>{globalize.translate(HeaderTitle)}</h2>
```

Comment:
Should we pass in the translated strings instead like we do for `SectionTitleContainer`?